packages/invoices/src/OutboundShipment/DetailView/modals/hooks/useNextItem.test.tsx

```diff
@@ -49,6 +49,12 @@ const getLines = (): InvoiceLineNode[] => [
       code: '',
       isVisible: true,
       name: '',
+      stats: {
+        __typename: 'ItemStatsNode',
+        stockOnHand: 0,
+        monthsOfStock: 0,
+        averageMonthlyConsumption: 0,
+      },
```

Comment:
these needed to be `ItemNode` 🙄  - I think really this hook should be using `ItemRowFragment`